dataland-frontend/src/components/forms/CreateSmeDataset.vue

```diff
@@ -121,11 +138,19 @@ import SingleSelectFormField from "@/components/forms/parts/fields/SingleSelectF
 import BigDecimalExtendedDataPointFormField from "@/components/forms/parts/fields/BigDecimalExtendedDataPointFormField.vue";
 import NaceCodeFormField from "@/components/forms/parts/fields/NaceCodeFormField.vue";
 import { type DocumentToUpload } from "@/utils/FileUploadUtils";
-import { objectDropNull, type ObjectType } from "@/utils/UpdateObjectUtils";
+import { type ObjectType } from "@/utils/UpdateObjectUtils";
 import { formatAxiosErrorMessage } from "@/utils/AxiosErrorMessageFormatter";
-import { getFilledKpis } from "@/utils/DataPoint";
 import { getBasePrivateFrameworkDefinition } from "@/frameworks/BasePrivateFrameworkRegistry";
 import { type PrivateFrameworkDataApi } from "@/utils/api/UnifiedFrameworkDataApi";
+import PollutionEmissionFormField from "@/components/forms/parts/fields/PollutionEmissionFormField.vue";
+import SubsidiaryFormField from "@/components/forms/parts/fields/SubsidiaryFormField.vue";
+import YesNoBaseDataPointFormField from "@/components/forms/parts/fields/YesNoBaseDataPointFormField.vue";
+import FreeTextFormField from "@/components/forms/parts/fields/FreeTextFormField.vue";
+import RadioButtonsFormField from "@/components/forms/parts/fields/RadioButtonsFormField.vue";
+import WasteClassificationFormField from "@/components/forms/parts/fields/WasteClassificationFormField.vue";
+import SiteAndAreaFormField from "@/components/forms/parts/fields/SiteAndAreaFormField.vue";
```

Comment:
some issues with this formField.
filling out (only) the required fields doesnt let me upload the report, even though im allowed to click the button.

Reply:
Can you check again, we have aligned the required fields in the frontend with the required fields in the backend.